frontend/src/main/components/EnrollmentData/EnrollmentHistoryGraph.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Legend,
+} from "recharts";
+
+// Stryker disable all: This is not to be run until the tests are written as it is a placeholder.
```

Comment:
Needs to be fixed, just like in #51 and #53

Reply:
This isn't a placeholder; this is your entire contribution.  And there's no "future PR"; this is the only one.  So the time to write tests is NOW.

Reply:
You need a file called `EnrollmentHistoryGraph.tests.js` that tests the functions in this file, and the component you are rendering.